stories/annotations/ellipse.stories.tsx

```diff
@@ -1,46 +1,48 @@
 import { Meta } from '@storybook/react';
-import React from 'react';
 
 import { Annotation } from '../../src';
+import { EllipseProps } from '../../src/components/Annotations/Ellipse';
 
 import AnnotationData from './annotation.data';
 
 export default {
   title: 'Docs/Annotations/Ellipse',
+  component: Annotation.Ellipse,
   decorators: [
     (Story) => (
       <AnnotationData>
         <Story />
       </AnnotationData>
     ),
   ],
+  args: {
+    style: {
+      fill: 'red',
+    },
+  },
 } as Meta;
 
-export function AnnotationEllipseStories() {
-  return (
-    <Annotation.Ellipse
-      cx="300"
-      cy="270"
-      rx="90"
-      ry="20"
-      style={{ fill: 'red' }}
-    />
-  );
+export function AnnotationEllipseStories(props: EllipseProps) {
+  return <Annotation.Ellipse {...props} />;
 }
 
 AnnotationEllipseStories.storyName = 'Annotation ellipse with string value';
+AnnotationEllipseStories.args = {
+  cx: '300',
+  cy: '270',
+  rx: '90',
+  ry: '20',
+};
 
-export function AnnotationEllipseValuesStories() {
-  return (
-    <Annotation.Ellipse
-      cx={2}
-      cy={12}
-      rx={1}
-      ry="20px"
-      style={{ fill: 'red' }}
-    />
-  );
+export function AnnotationEllipseValuesStories(props: EllipseProps) {
+  return <Annotation.Ellipse {...props} />;
 }
 
 AnnotationEllipseValuesStories.storyName =
   'Annotation ellipse with number value';
+AnnotationEllipseValuesStories.args = {
+  cx: 2,
+  cy: 12,
+  rx: 1,
+  ry: 1,
```

Comment:
I think there's a bug to fix:
![image](https://user-images.githubusercontent.com/2352663/111296953-5e710100-864d-11eb-9d29-210b52f6cb2b.png)